ansible/eda-server
# Changes activation foreign key for activation instances to not nullable.

### Jira Ticket: [AAP-11605](https://issues.redhat.com/browse/AAP-11605) ###

### Purpose: ###
- To validate that the websocket url is valid and connecting before the creation of an activation
- also changed decision environment to not nullable since they are required for running activations

### Testing Instructions: ###
1. Add a project, decision environment, and controller token
2. Create an activation and see that it starts
3. in settings change the websocket base URL to something invalid run `task dev:init`
4. Try and create the activation again and you should get a 422
